GliaWidgets/Lib/View/View.swift

```diff
@@ -14,10 +14,26 @@ class View: UIView {
 
     init() {
         super.init(frame: .zero)
+        setup()
     }
 
     @available(*, unavailable)
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    override func updateConstraints() {
```

Comment:
Why not call defineLayout after setup in initialiser and get rid of the needsDefineLayout variable?

Reply:
Because of creation constraints, it is part of establishing UI and it is not required to do it on construction/initialization step. Ex. you can just create a view and keep it in memory for a long time.